doc/interfaces_overview.md

```diff
@@ -35,21 +35,21 @@ r.slope.aspect elevation=elevation slope=slope aspect=aspect
 
 ## Python
 
-The `grass.script` module provides a Python interface to GRASS. This allows
-users to write Python scripts to interact with GRASS. The `grass.script` module
-contains the `gs.script.core` module which provides the core functionality for the
-GRASS Python interface, the `grass.script.raster` module which provides
-functionality for working with raster data, and the `grass.script.vector` module
-which provides functionality for working with vector data.
-
-To get started Python, create a new project with `gs.create_project` and start a
-GRASS session with the `grass.script.setup.init` function to initialize the
+GRASS Python interface provides libraries to create GRASS scripts and access
+the internal data structures of GRASS. The Python interface consists of
+two main libraries:
+*[grass.script](https://grass.osgeo.org/grass-stable/manuals/libpython/script_intro.html)*
+provides a Python interface to GRASS tools
+and *[grass.pygrass](https://grass.osgeo.org/grass-stable/manuals/libpython/pygrass_index.html)*
+enables access to the internal data structures of GRASS.
+
+To get started with scripting, create a new project with `gs.create_project` and
+start a GRASS session with the `gs.init` function to initialize the
```

Comment:
We don't need to do it that way, but init is still used through _setup_, it just does not require additional import as in the past.

```suggestion
start a GRASS session with the `gs.setup.init` function to initialize the
```

See: https://github.com/OSGeo/grass/blob/main/python/grass/script/__init__.py#L3